js/script.js

```diff
@@ -1 +1,37 @@
- 
+var prefix = "https://cors-anywhere.herokuapp.com/";
+var tweetLink = "https://twitter.com/intent/tweet?text="; 
+var quoteUrl = "https://quotesondesign.com/wp-json/posts?filter[orderby]=rand&filter[posts_per_page]=1";
+
+function getQuote() { 
+    $.getJSON(prefix + quoteUrl, createTweet);
+    $.ajaxSetup({ cache: false });
+}
+
+function createTweet(input) { 
+    var data = input[0]; 
```

Comment:
you select fist element of input - at first check array input has at least one element :)